ruby/private/toolchains/ruby_runtime.bzl

```diff
@@ -88,14 +88,16 @@ def _install_ruby(ctx, ruby):
 
 def host_ruby_is_correct_version(ctx, version):
     interpreter_path = ctx.which("ruby")
+
     if not interpreter_path:
+        print("Can't find ruby interpreter in the PATH")
         return False
 
-    ruby_version = ctx.execute(["ruby", "--version"]).stdout
-    version_string = "ruby %sp" % version
+    ruby_version = ctx.execute(["ruby", "-e", "print RUBY_VERSION"]).stdout
+
+    print("Found existing Ruby, local version is '%s', required version is '%s'" % (ruby_version, version))
```

Comment:
Can we move this into an if statement that only prints if the local version is different than required?

Reply:
Done.